src/main/app/src/app/shared/material-form-builder/model/abstract-form-field-builder.ts

```diff
@@ -51,7 +51,7 @@ export abstract class AbstractFormFieldBuilder {
         return this;
     }
 
```

Comment:
kan je hier niet gewoon AbstractFormField zeggen ipv this['formField']

Reply:
Nee, dit is om er voor te zorgen dat de classes die van de abstractformfield extenden hun eigen implementatie van formfield terug geven, waardoor je niet meer hoeft te casten naar de instantie van het formfield